packages/@react-aria/tag/docs/useTagGroup.mdx

```diff
@@ -0,0 +1,296 @@
+    ref
+  );
+
+  return (
+    <span
+      ref={ref}
+      {...rowProps}
+      {...focusProps}
+    >
+      <span {...gridCellProps} {...labelProps}>{item.rendered}</span>
+      {allowsRemoving && 
+        <Button {...clearButtonProps}>❎</Button>}
+    </span>
+  );
+}
+
+<TagGroup label="Categories">
+  <Item key="news">News</Item>
+  <Item key="travel">Travel</Item>
+  <Item key="gaming">Gaming</Item>
+  <Item key="shopping">Shopping</Item>
+</TagGroup>
+```
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show CSS</summary>
+
+```css
+/* css */
+.tag-group {
+  display: flex;
+  flex-direction: column;
+}
+
+.tag-group div {
+  margin: 5px 0;
+}
+
+.tag-group [role="grid"] {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.tag-group [role="row"] {
+  display: flex;
+  align-items: center;
+  border: 1px solid gray;
+  border-radius: 4px;
+  padding: 2px 5px;
+  margin: 2px;
+}
+
+.tag-group [role="gridcell"] {
+  margin: 0 5px;
+}
+
+.tag-group [role="row"] button {
+  background: none;
+  border: none;
+}
+
+.tag-group .description {
+  font-size: 12px;
+}
+
+.tag-group .error-message {
+  color: red;
+  font-size: 12px;
+}
+```
+
+</details>
+
+### Button
+
+The `Button` component is used in the above example to remove a tag. It is built using the [useButton](useButton.html) hook, and can be shared with many other components.
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
+
+```tsx example export=true render=false
+import {useButton} from '@react-aria/button';
+
+function Button(props) {
+  let ref = React.useRef(null);
+  let {buttonProps} = useButton(props, ref);
+  return <button {...buttonProps} ref={ref}>{props.children}</button>;
+}
+```
+
+</details>
+
+
+## Styled examples
+
+<ExampleCard
+  url="https://codesandbox.io/s/usetaggroup-with-tailwind-css-zxxrpv"
+  preview={tailwindExample}
+  title="Tailwind CSS"
+  description="A TagGroup built with Tailwind." />
+
+## Usage
+
+### Remove tags
+
+The `onRemove` handler and `allowsRemoving` props can be used to include a clear button which can be used to remove a tag. This allows the user to press the clear button, or press the backspace key while the tag is focused to remove the tag from the group.
+
+```tsx example
+import {useListData} from '@react-stately/data';
+
+function Example() {
+  let list = useListData({
+    initialItems: [
+      { id: 1, name: "News" },
+      { id: 2, name: "Travel" },
+      { id: 3, name: "Gaming" },
+      { id: 4, name: "Shopping" }
+    ]
+  });
+
+  return (
+    <TagGroup
+      label="Categories"
+      items={list.items}
+      onRemove={list.remove}
+      allowsRemoving>
+      {(item) => <Item>{item.name}</Item>}
+    </TagGroup>
+  );
+}
+```
+
+### Description
+
+The `description` prop can be used to associate additional help text with a tag group.
+
+```tsx example
+<TagGroup label="Categories" description="Select your desired categories.">
```

Comment:
I feel like 'Your selected categories' would be better. There is nothing interactive where you can perform a selection right now, so this wording is a bit odd. Same thing in the error message case. Maybe 'Invalid set of categories'? just to reinforce that we didn't do a selection?